python/tvm/topi/nn/conv2d.py

```diff
@@ -382,7 +382,15 @@ def conv2d_nhwc(
     if auto_scheduler_rewritten_layout:
         # Infer shape for the rewritten layout
         # todo(merrymercy): wrap this with a more general interface.
-        if len(Filter.shape) >= 10:
+        if len(Filter.shape) == 17:
```

Comment:
Could you maybe add some comments about the constants?

Reply:
Yes, I will supply some comment about why we get this.